src/components/Text/Text.tsx

```diff
@@ -41,8 +41,8 @@ class Text extends UIComponent<Extendable<ITextProps>, any> {
     /** Change the default element type of the Text component */
     as: customPropTypes.as,
 
-    /** Set as @mention Text component */
-    atMention: PropTypes.bool,
+    /** An @mention Text component can be formatted for myself mentioned or an other person mentioned */
```

Comment:
Phrasing is a bit odd here.  There is no "@mention Text component".  There is a "Text component" that can be used to show an "at mention".  One potential rewrite here might be:

```jsx
/**
 * At mentions can be formatted draw users' attention.
 * Mentions for the current user can be formatted differently than mentions for other names.
 */
```